backend/helpers.js

```diff
@@ -0,0 +1,10 @@
+import { string, object, boolean } from 'yup';
+
+export const validateValue = (body) => {
+    let requiredFields = object({
+        text: string().required(),
+        is_checked: boolean().required()
+      })
```

Comment:
This schema does not change and so can be moved outside the function and then reused